configure.py

```diff
@@ -1356,7 +1356,7 @@ def configure_zos(o):
   o['variables']['node_static_zoslib'] = b(True)
   if options.static_zoslib_gyp:
     # Apply to all Node.js components for now
-    o['variables']['zoslib_include_dir'] = Path(options.static_zoslib_gyp).parent + '/include'
+    o['variables']['zoslib_include_dir'] = Path(options.static_zoslib_gyp).parent / '/include'
```

Comment:
https://docs.python.org/3/library/pathlib.html#pathlib.Path
>  If the argument is an absolute path, the previous path is ignored. 

Which means this would end up being `/include` instead of the `include` path being joined to the parent?

Reply:
Good catch, changing it from `/include` to `include` fixes it, I'll apply this change.